### ISSUE TITLE
Structure of Finance for IT section

### ISSUE THREAD

**Issue by cioportfolio** [dm-academy/aitm#42]
Charles,

I think it might be worth attacking this topic is a slightly different way. When I am introducing people to these topics I usually approach it in stages:
- General financial principles (capital asset accounting, fixed and variable costs, budgeting etc)
- Business services topics which apply to HR, Finance etc and not just IT (business planning, provisioning, recharges, governance etc)
- Change and investment which apply generally (benefits realisation, risk, return on investment, project finance etc)
- IT specifics (no IT projects, no direct IT benefits, lack of maturity around IT assets)
- Discussion topics (black swan risks, agile, gap between memes and accounting reality such as IT as profit centre).

The first three might be best handled through references to generic material. You could argue that students should not get too far into the latter two areas until they have a proper understanding of the basics which are not IT specific.

**Comment by CharlesTBetz** [comment]
Yes, it was s dilemma how deeply to go into the non-IT foundations. Same issues with HR and sourcing to some degree. I'm going to leave this issue open as guidance for rewrite, may take some time. Thanks!
